uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -210,4 +202,46 @@ public void activateAdmin() {
             Assert.fail(e.getMessage());
         }
     }
+
+    //obtaining date button in different devices.
+    UiObject changeDateButton()
+    {
+        if(android.os.Build.VERSION.SDK_INT == 28)
+            return UiAutomatorUtils.obtainUiObjectWithText("Set date");
+
+        return UiAutomatorUtils.obtainEnabledUiObjectWithExactText("Date");
+    }
+
+    //obtaining button in a scrollable in different devices.
+    UiObject obtainButtonInScrollable(String Text)
+    {
+        if(android.os.Build.VERSION.SDK_INT == 28)
+            return UiAutomatorUtils.obtainChildInScrollable(
+                    "com.android.settings:id/list",
+                    Text
+            );
+
+        return UiAutomatorUtils.obtainChildInScrollable(
+                "com.android.settings:id/recycler_view",
+                Text
+        );
+    }
+
+    //obtaining disable admin button in different devices.
+    UiObject disableAdminButton(final DeviceAdmin deviceAdmin)
+    {
+        if(android.os.Build.VERSION.SDK_INT == 28)
+            return UiAutomatorUtils.obtainChildInScrollable(
+                    "android:id/list",
+                    deviceAdmin.getAdminName()
+            );
+
+        return UiAutomatorUtils.obtainChildInScrollable(
+                "com.android.settings:id/recycler_view",
+                deviceAdmin.getAdminName()
+        );
+    }
```

Comment:
The code formatting doesn't match our current code style. Please follow the same code style as the one being used throughout the rest of the code base.

Reply:
reformated the code.

Reply:
Please follow same code styles (braces etc..) as the one being used right now